weepickle/test/src-jvm/com/rallyhealth/weepickle/v1/example/JvmPrimitiveTests.scala

```diff
@@ -16,6 +16,7 @@ object JvmPrimitiveTests extends TestSuite {
       test("LocalTime") { rw(LocalTime.parse("00:00"), "\"00:00\"") }
       test("LocalDateTime") { rw(LocalDateTime.parse("2000-01-01T00:00:00"), "\"2000-01-01T00:00:00\"") }
       test("OffsetDateTime") { rw(OffsetDateTime.parse("2007-12-03T10:15:30+01:00"), "\"2007-12-03T10:15:30+01:00\"") }
+      test("ZonedDateTime") {rw(ZonedDateTime.parse("2000-01-01T00:00:00Z"), "\"2000-01-01T00:00:00Z\"") }
```

Comment:
should we test an actual zone?

Reply:
You mean one that isn't the same as UTC? We can I guess, but I don't see the difference.

Reply:
pretty easy to add though? can  I suggest PST :D

Reply:
Agreed -- UTC is kind of a special case, so I'd recommend adding one other zone as a belt-and-suspenders check.  Other than that, this looks good...

Reply:
But it wouldn't be PST, it would be something like "2000-01-01T00:00:00-08:00[America/Los_Angeles]"

Reply:
Fair enough, swapped in Central time